src/main/java/org/broadinstitute/hellbender/utils/fragments/FragmentUtils.java

```diff
@@ -45,9 +45,9 @@ public static void adjustQualsOfOverlappingPairedFragments(final GATKRead clippe
         final int numOverlappingBases = Math.min(clippedFirstRead.getLength() - firstReadStop, clippedSecondRead.getLength());
 
         final byte[] firstReadBases = clippedFirstRead.getBases();
-        final byte[] firstReadQuals = clippedFirstRead.getBaseQualities();
+        final byte[] firstReadQuals = clippedFirstRead.getBaseQualities().clone();
```

Comment:
`GATKRead.getBases()` and `GATKRead.getBaseQualities()` are safe methods and return a copy of the array. Thus, it is not really modified in place. I think that this patch is not needed for that reason.